internal/pkg/cli/env_init.go

```diff
@@ -56,6 +56,7 @@ https://aws.github.io/copilot-cli/docs/credentials/#environment-credentials`
 	envInitDefaultRegionOption = "us-west-2"
 
 	fmtDeployEnvStart        = "Proposing infrastructure changes for the %s environment."
+	fmtDeployEnvSuccess      = "Proposed infrastructure changes for the %s environment."
```

Comment:
Why did we add this? I think it was intentional so that the line:
`"Proposing infrastructure changes for the %s environment."`  gets overriden and becomes:
`"Creating the infrastructure for the %s environment."`

Reply:
Because on Windows the workflow is not like this: `"Creating the infrastructure for the %s environment."` is at the bottom instead of above those progress info. And `"Proposing infrastructure changes for the %s environment."` will just disappear and leave it to be an empty line. But I'm ok if we want to remove it.